ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -2296,3 +2296,43 @@
 #                                   f"{str(fixed_sheet_row_data)} : {str(formatted_orders[index])}")
 #             for item in formatted_orders[index]:
 #                 self.assertIn(item, fixed_sheet_row_data)
+
+    def test071_create_suborders_same_testunit(self):
+        """
+        Create 5 suborders with same test units ( single select ) and make sure 5 analysis
+        records created successfully according to that.
+
+        LIMS-4249
+        """
+        response, payload = self.orders_api.create_new_order()
```

Comment:
in test case ticket on Jira, step 3 it says press on create button . So you need to create order using GUI not API.
You can get random trst unit using api and get its material type.